modules/SofaGuiQt/src/sofa/gui/qt/viewer/qgl/QtGLViewer.cpp

```diff
@@ -868,7 +868,6 @@ void QtGLViewer::keyPressEvent ( QKeyEvent * e )
         default:
         {
             SofaViewer::keyPressEvent(e);
-            QGLViewer::keyPressEvent(e);
```

Comment:
ok but are you sure both viewer were doing the same thing?

Reply:
Checking again, SofaViewer doesn't inherite from QGLViewer... so you might loose some QGLViewer key commands. 
Are they important ? I let you decide.

Reply:
ok!